mountpoint-s3-client/CHANGELOG.md

```diff
@@ -27,8 +27,12 @@
   ([#1192](https://github.com/awslabs/mountpoint-s3/pull/1192))
 
 ### Other changes
-
-* `HeadObjectResult` now includes the server-side encryption settings used when storing the object.
+* Consume extension for Copy Operation, that required extra header with the Source URI [aws/aws-c-s3#482](https://github.com/awslabs/aws-c-s3/pull/482)
+* Auto - Update S3 Ruleset & Partition [aws/aws-c-s3#483](https://github.com/awslabs/aws-c-s3/pull/483)
+* Add retry on failed TLS negotiation [aws/aws-c-s3#478](https://github.com/awslabs/aws-c-s3/pull/478)
+* Add retry on ExpiredToken [aws/aws-c-s3#472](https://github.com/awslabs/aws-c-s3/pull/472)
+* Respect checksum header over settings from options [aws/aws-c-s3#474](https://github.com/awslabs/aws-c-s3/pull/474)
```

Comment:
I'd revert this. We do not usually report the changes in `aws-c-s3` here, unless they fix a known issue in Mountpoint/client.

Reply:
The change to `Copy` does not affect us, yet. I expect #1228 will add an entry to the changelog to describe what it fixes.